package/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.13 AS helm
-RUN git -C / clone --branch v3.3.3-rancher3 --depth=1 https://github.com/rancher/helm
+FROM golang:1.16 AS helm
```

Comment:
I would prefer this gets pinged to 1.16.4 so we know which exact version we are putting.